src/components/Login/LoginContainer.tsx

```diff
@@ -188,12 +187,9 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
 
       if (queryEndpoint && queryKey) {
         if (!RegExp(DOMAIN_FORMAT).test(queryEndpoint)) {
-          const error = new Error(`Invalid endpoint: ${queryEndpoint}`);
+          const error = new SessionError(`Invalid endpoint: ${queryEndpoint}`);
```

Comment:
Hmm why is this a session error?

Reply:
Good catch, will refactor it back to:
```suggestion
          const error = new Error(`Invalid endpoint: ${queryEndpoint}`);
```